### PR TITLE
Fixed the date issue

### PR DESCRIPTION
**Pull request by andrewtran30**

### CODE REVIEW

File: src/components/Time/index.tsx [hunk 1]
```diff
@@ -13,8 +13,8 @@ var event = new Event('timestamp')
 let minute: number
 
 setInterval(() => {
-  const date: Date = new Date()
-  const currentMinutes: number = date.getMinutes()
+  var date: Date = new Date()
+  var currentMinutes: number = date.getMinutes()
```
Comment (mitchellsimoens) [inline src/components/Time/index.tsx]: Any reason you changed these from `const` to `var`? It's normally preferred to use `const` unless you need to change the variable assignment then `let` is more preferred. I don't remember last time I've actually used `var` :)

File: src/service/Data.tsx [hunk 2]
```diff
@@ -118,8 +118,8 @@ const processTimestamps = (records: Modite[] = [], date: Date) => {
   const nowUtc: Date = new Date(date.getTime() + date.getTimezoneOffset())
 
   records.forEach((item: Modite) => {
-    const itemDate: Date = new Date((nowUtc as any) - (item.tz_offset as number) * 60000)
-
+    var CAoffset = -3
```
Comment (mitchellsimoens) [inline src/service/Data.tsx]: Another `var`

File: src/service/Data.tsx [hunk 3]
```diff
@@ -118,8 +118,8 @@ const processTimestamps = (records: Modite[] = [], date: Date) => {
...
+    const itemDate: Date = new Date(nowUtc as any)
```
Comment (mitchellsimoens) [inline src/service/Data.tsx]: With typescript, `any` is almost always considered bad. You have explicitly set it here which is better than implicit but you should be more specific of what `nowUtc` is. Looks like it is defined on line 118 and it's a `Date` so should replace `any` with `Date`.

File: api/modites.js [hunk 4]
```diff
@@ -41,7 +41,7 @@ const appendModiteProfiles = async modites => {
 
   const profileFetches = modites.map(modite => {
     return new Promise(async resolve => {
-      const profileObj = await MODITES.get(modite.id, 'json')
+      const profileObj = await modites.get(modite.id, 'json')
```
Comment (grgur) [inline api/modites.js]: this is a different thing. They are intentionally UPPERCASE because the values are stored in the Key-Value database on Cloudflare

**Comment by grgur** [comment]
Automation files shouldn't be deleted

**Pull request closed without merging**